docs/source/_json/search.json

```diff
@@ -19,5 +25,5 @@ content-type: application/json
       "title": "Test Folder"
     }
   ], 
-  "items_count": 2
+  "total_items": 2
```

Comment:
We could, I don't have any strong feelings about either one ;) Regarding developer-friendly representations though, we might want to look into using `OrderedDict`s at some point. If given an ordered dict, `json.dumps()` will preserve the order of the keys. Structurally the order of keys in a dict still has no meaning of course, but that might make it easier to produce JSON that is layed out in a human-friendly way.